conftest.py

```diff
@@ -16,7 +16,7 @@
 collect_ignore = _get_doctest_blacklist()
 
 # Set up printing for doctests
-setup_pprint()
+setup_pprint(disable_line_wrap=False)
```

Comment:
Presumably pytest-doctestplus would actually do the right thing without this flag, but this also breaks the line pprint tests that test the line wrapping defaults.